src/File/Upload.php

```diff
@@ -109,7 +110,7 @@ public function getFiles($file = null)
      */
     public function setFiles($files = [])
     {
-        if (count($files) === 0) {
+        if ((is_array($files) || $files instanceof Countable) && count($files) === 0) {
```

Comment:
#### Side note
This bug highlights a higher priority bug in this method: the argument type isn't checked.
But to lessen the complexity of the PR and to speed up the next release, here I'm doing the minimum to get the code to work **exactly** like before.